packages/react-components/react-breadcrumb/src/components/Breadcrumb/useBreadcrumb.ts

```diff
@@ -12,19 +12,18 @@ import type { BreadcrumbProps, BreadcrumbState } from './Breadcrumb.types';
  * @param ref - reference to root HTMLElement of Breadcrumb
  */
 export const useBreadcrumb_unstable = (props: BreadcrumbProps, ref: React.Ref<HTMLElement>): BreadcrumbState => {
+  const { size, dividerType, ...rest } = props;
   return {
-    // TODO add appropriate props/defaults
     components: {
-      // TODO add each slot's element type or component
       root: 'nav',
       list: 'ol',
     },
-    // TODO add appropriate slots, for example:
-    // mySlot: resolveShorthand(props.mySlot),
     root: getNativeElementProps('nav', {
       ref,
-      ...props,
+      ...rest,
     }),
     list: {},
```

Comment:
```suggestion
    list: resolveShorthand(props.list, { required: true }),
```

You forgot to use `resolveShorthand()` to resolve a slot from `props`. (You can do this in a follow up PR)